Sami/Resources/themes/default/sami.js.twig

```diff
@@ -1,5 +1,9 @@
 {% from _self import element %}
 
+{% block project_version %}
```

Comment:
I don't see anywhere, that you are rendering this block.

Either this is some kind of magical block name, that is rendered in any case OR you just forgot to render it.